db/migrate/20220706163605_fix_comment_counter_name.rb

```diff
@@ -0,0 +1,6 @@
+class FixCommentCounterName < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :posts, :comment_counter, :comments_counter
+    rename_column :users, :post_counter, :posts_counter
+  end
+end
```

Comment:
- [ ] Please check the migration files, when I am running rails db:migrate I am getting errors. perhaps you deleted some migrations manually. To reproduce the error you have to drop the DB and run migrations again. 

For this reason I cannot run the tests, they are also failing. 

<img width="901" alt="Screenshot 2022-07-19 at 13 52 59" src="https://user-images.githubusercontent.com/77038610/179722571-6eecf023-ad32-450a-841e-eebacb52a233.png">

Reply:
Hey @ShoiraTa thank you for this review , be assured this has been worked on.